### ISSUE TITLE
duplicate message

### ISSUE THREAD

**Issue by sd-timaru** [fluttercommunity/flutter_sms#17]
Hi!
In Android 8, the message that arrives at the sms application is duplicated (it appears twice the initially set message).
On Android 9 and 10 is ok.

It may have something to do with putting twice the message in the intent.putExtra:

``` 
private fun sendSMS(result: Result, phones: String?, message: String?) {
    val intent = Intent(Intent.ACTION_SENDTO)
    intent.data = Uri.parse("smsto:$phones")
    intent.putExtra("sms_body", message)
    intent.putExtra(Intent.EXTRA_TEXT, message)
    activity?.startActivityForResult(intent, REQUEST_CODE_SEND_SMS)

  }

**Comment by rodydavis** [comment]
Please feel free to submit a PR, I haven't experienced this

**Comment by Elykauf** [comment]
Experienced the same thing. Maybe add a conditional checking for the version of android.

**Comment by rodydavis** [comment]
fixed on version `2.3.1` reopen if it is still an issue
